pyproject.toml

```diff
@@ -1,3 +1,62 @@
+[project]
+name = "fxci-config"
+version = "1.0.0"
+description = "Administration of runtime configuration (Taskcluster settings) for Firefox CI"
+authors = [
+    {name = "Mozilla Release Engineering", email = "release+ciadmin@mozilla.com"}
+]
+readme = "README.md"
+requires-python = ">=3.11"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.11",
+]
+dependencies = [
+    "tc-admin>=4.0.2",
+    "taskcluster",
+    "attrs",
+    "mozilla-repo-urls",
+    "aiohttp[speedups]<4",
+    "pyyaml>=5.3",
+    "iso8601==0.1.12",
+    "json-e",
+    "aiohttp-retry",
+    "simple-github>=2.2.0",
+    # These are used at run-time as part of `tc-admin check`
+    "pytest",
+    "pytest-asyncio",
+    "async-timeout",
+]
+
+[dependency-groups]
+dev = [
+    "coverage[toml]",
+    "flake8",
+    "google-cloud-compute",
+    "pip>=21.2",
+    "pytest",
+    "pytest-cov",
+    "pytest-mock",
+    "pytest-taskgraph>=0.1.0",
+    "responses>=0.25.3",
+    "taskcluster-taskgraph>=13.0.0",
+    "tox",
+]
+
+[project.scripts]
+ci-admin = "ciadmin.boot:boot"
+fxci = "fxci.cli:main"
+
+[tool.uv.workspace]
+members = ["build-decision"]
+
+[tool.uv.build-backend]
+module-name = ["ciadmin", "fxci"]
+
+[build-system]
```

Comment:
Uh, this is new to me. Why are we not using hatchling here like in every other project? (i.e https://github.com/mozilla-releng/balrog/blob/0cda941f138903cdff94e2a3cbc5b27fbe90611d/pyproject.toml#L71-L76)

Reply:
I don't have strong feelings about it, I didn't even know it was a thing tbqh. I'm fine if that's a conscious choice because it's "better" and not "oops there were 2 alternatives and I picked the one we don't use".

I would probably leave it unpinned though so it matches the uv version?

Reply:
Yeah sure, it's probably not technically best practice, but our build is so simple that I highly doubt anything will ever break.. and if it does it can be fixed easily enough.